src/current/_includes/releases/v25.4/v25.4.0.md

```diff
@@ -0,0 +1,213 @@
+   <td>
+    <p class="feature-summary">Transaction Diagnostic Bundles for Comprehensive SQL Tracing</p>
+    <p class="feature-description" markdown="1">
+      CockroachDB now supports Transaction Diagnostic Bundles, enabling Cockroach Labs Support to capture and analyze a complete trace of an entire SQL transaction not just individual statements.
+
+      This feature extends the existing Statement Diagnostic Bundle capability to provide a holistic view of multi-statement transactions, making it significantly easier to diagnose complex customer issues during support incidents.
+    </p>
+    <p class="feature-description">Each Transaction Diagnostic Bundle includes:</p>
+    <ul class="feature-description">
+      <li>A Jaeger and text trace of the full transaction execution path.</li>
+      <li>Individual statement diagnostic bundles for every statement executed within the transaction.</li>
+    </ul>
+   </td>
+   <td>25.4</td>
+   <td class="icon-center">{% include icon-yes.html %}</td>
+   <td class="icon-center">{% include icon-yes.html %}</td>
+   <td class="icon-center">{% include icon-no.html %}</td>
+   <td class="icon-center">{% include icon-no.html %}</td>
+  </tr>
+  <tr>
+   <td>
+    <p class="feature-summary">Structured and Consistent Logging Channels</p>
+    <p class="feature-description" markdown="1">
+      CockroachDB has updated structured and consistent logging channels across the system to improve log discoverability, production observability, and support troubleshooting.
+    </p>
+    <p class="feature-description">New channels introduced:</p>
+    <ul class="feature-description">
+      <li>CHANGEFEED – dedicated channel for Change Data Capture (CDC) logs, migrated from DEV and TELEMETRY.</li>
+      <li>KV_EXEC – channel for KV execution logs, separated from general distribution logs.</li>
+    </ul>
+    <p class="feature-description">Rerouted existing logs:</p>
+    <ul class="feature-description">
+      <li>sampled_query logs moved from TELEMETRY → SQL_EXEC for better categorization.</li>
+    </ul>
+   </td>
+   <td>25.4</td>
+   <td class="icon-center">{% include icon-yes.html %}</td>
+   <td class="icon-center">{% include icon-no.html %}</td>
+   <td class="icon-center">{% include icon-no.html %}</td>
+   <td class="icon-center">{% include icon-no.html %}</td>
+  </tr>
+ </tbody>
+</table>
+
+<h4 id="v25-4-0-cloud">CockroachDB Cloud</h4>
+
+<table markdown="1">
+ <thead>
+  <tr>
+   <th class="center-align" colspan="1" rowspan="2">Feature</th>
+   <th class="center-align" colspan="5" rowspan="1">Availability</th>
+  </tr>
+  <tr>
+    <th colspan="1" rowspan="1">Ver.</th>
+    <th colspan="1" rowspan="1" style="white-space: nowrap;">Self-hosted</th>
+    <th colspan="1" rowspan="1">Advanced</th>
+    <th colspan="1" rowspan="1">Standard</th>
+    <th colspan="1" rowspan="1">Basic</th>
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td>
+    <p class="feature-summary">Metrics Viewer Role for CockroachDB Cloud</p>
```

Comment:
@florence-crl Please review this 1 cloud feature highlight. It's observability-related and I know you're familiar with it. Thank you.